src/components/common/ColonyHome/partials/Agreements/Agreements.tsx

```diff
@@ -0,0 +1,73 @@
+import clsx from 'clsx';
+import React from 'react';
+
+import { ACTION } from '~constants/actions';
+import { useActionSidebarContext } from '~context/ActionSidebarContext';
+import { COLONY_AGREEMENTS_ROUTE } from '~routes';
+import { formatText } from '~utils/intl';
+import { ACTION_TYPE_FIELD_NAME } from '~v5/common/ActionSidebar/consts';
+import WidgetBox from '~v5/common/WidgetBox';
+import EmptyWidgetState from '~v5/common/WidgetBox/partials';
+import MessageNumber from '~v5/shared/MessageNumber';
+import TitleWithNumber from '~v5/shared/TitleWithNumber';
+
+const displayName = 'common.ColonyHome.TotalActions';
+
+const Agreements = () => {
+  const agreements = undefined;
```

Comment:
This is supposed to be connected later? Could you add a `TODO` comment indicating that if that's the case?